assets/scss/bootstrap/_wells.scss

```diff
@@ -0,0 +1,29 @@
+//
+// Wells
+// --------------------------------------------------
+
+
+// Base class
+.well {
+  min-height: 20px;
+  padding: 19px;
+  margin-bottom: 20px;
+  background-color: $well-bg;
+  border: 1px solid $well-border;
+  border-radius: $border-radius-base;
+  @include box-shadow(inset 0 1px 1px rgba(0,0,0,.05));
+  blockquote {
+    border-color: #ddd;
+    border-color: rgba(0,0,0,.15);
```

Comment:
Color literals like `rgba(0, 0, 0, 0.15)` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Property `border-color` already defined on line 16<br>`.15` should be written with a leading zero as `0.15`<br>Commas in function arguments should be followed by a single space